contributing/code/standards.rst

```diff
@@ -5,7 +5,7 @@ Symfony code is contributed by thousands of developers around the world. To make
 every piece of code look and feel familiar, Symfony defines some coding standards
 that all contributions must follow.
 
-These Symfony coding standards are based on the `PSR-1`_, `PSR-2`_ and `PSR-4`_
+These Symfony coding standards are based on the `PSR-1`_, `PSR-12`_ and `PSR-4`_
```

Comment:
PSR-12 builds on top op PSR-2, just like PSR-2 builds on top of PSR-1. So we should probably keep PSR-2.

Reply:
The link to PSR-12 is missing right now.

Reply:
Fixed while merging. Thanks!